app/src/main/java/by/itechart/android/ui/entity/DialogUIModel.kt

```diff
@@ -1,7 +1,7 @@
 package by.itechart.android.ui.entity
 
 
-class DialogUIModel(
+data class DialogUIModel(
```

Comment:
data class DialogUIModel(
    val type: Type,
    val message: String,
    val title: String? = null,
    val acceptText: String? = null,
    val declineText: String? = null,
    val icon: Int? = null
) {
    enum class Type { ACCEPT, INFO, BOTTOM }
}

or move enum to separate file